nncf/config/config.py

```diff
@@ -67,6 +65,28 @@ def has_extra_struct(self, struct_cls: Type[NNCFExtraConfigStruct]) -> NNCFExtra
     def get_all_extra_structs_for_copy(self) -> List[NNCFExtraConfigStruct]:
         return list(self.__nncf_extra_structs.values())
 
+    def get_redefinable_global_param_value_for_algo(self, param_name: str, algo_name: str) -> Optional:
```

Comment:
I have concerns about the usage scenario of this method. I mean "Some parameters" say me that I don't know when I should use this method. It will be more clear if I get all attributes using this logic or this logic will be included in the extract functionality. For example, into `extract_algo_specific_config`.

Reply:
Sadly, the exact names of the parameters that are redefinable are algo-specific, while `extract_algo_specific_config` is currently a dumb function that only parses the most basic schema-guaranteed config structure.